File: Day 14/Aayushi-Mittal.py
# To find whether a number is Perfect or not.
# Perfect number is a positive integer which is equal to the sum of its proper positive divisors.

num=0 
sum=0;
num = int(input("enter a number : "))
for i in range(0, num):
    if num%i==0:
        sum+=i
if sum==num:
    print(sum," is a perfect number.")
else:
    print(num, " is not a perfect number.")

File: Day 14/Aayushi-Mittal.cpp
// To find whether a number is Perfect or not.

#include <iostream>
#include<math.h>
using namespace std;
int main()
{
    int num=0, i, sum=0;
    cout<<"enter a number : "<<num<<endl;
    cin>>num;
    for(i=1; i<num; i++)
    {
        if(num%i==0)
            sum+=i;
    }
    if (sum==num)
      cout<<sum<<" is a perfect number."<<endl;
    else
      cout<<num<<" is not a perfect number."<<endl;
    return 0;
}
